spec/factories/profile_link_sites.rb

```diff
@@ -131,6 +131,60 @@
       validate_find '\A(https?://)?(www.)?(patreon.com/)?(?<username>[a-zA-Z0-9_\-]+)\z'
       validate_replace 'https://patreon.com/\k<username>'
     end
+
+    # DeviantArt
+    trait :deviantart do
+      validate_find '\A(https?://)?(www.)?(?<username>[a-zA-z0-9_\-]+)(.deviantart.com)?\z'
```

Comment:
You can just throw a rubocop disable/enable metrics to all of these, we don't care if they go over the line limit.